src/components/ImageWithSideTitle/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const ImageWithSideText = ({ imgUrl, title }) => {
+  return (
+    <div className="flex justify-around bg-c700 rounded py-2 hover:bg-c100 hover:opacity-75">
```

Comment:
why you used flex here!

- avoid using flex if it's not necessary
- give it specific padding to be like the design
- background and color should back to the component as a prop, and cover all the design colors